packages/dual-channel/src/app/components/text/index.js

```diff
@@ -53,7 +53,14 @@ const Chapter = styled.section`
   }
 `
 
-const Body = BaseComponents.ArticleContainer
+const Body = styled(BaseComponents.ArticleContainer)`
+  &::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 1px;
+  }
+`
```

Comment:
Why do we need to add 1px height pseudo element here ?

Reply:
Sorry for late reply.

This pseudo element makes `Body`'s height includes `margin-bottom` of last text section.
See the following screenshots for more details.

<img width="1280" alt="Screen Shot 2021-01-03 at 2 30 29 PM" src="https://user-images.githubusercontent.com/3000343/103473173-926bb280-4dd0-11eb-85e0-5b9c05a8a3bb.png">

_Screenshot 1: Body with pseudo element `&::after`_

<img width="1280" alt="Screen Shot 2021-01-03 at 2 30 40 PM" src="https://user-images.githubusercontent.com/3000343/103473175-9566a300-4dd0-11eb-9be4-c5ce0170bb4b.png">

_Screenshot 2: Body without pseudo element `&::after`_

Because `Body`'s height includes the last margin, the last embedded element will be precisely `position: absolute` at the bottom.

Reply:
Thanks for quite a clear elaboration.